migrations/20231007120000_create_tiny_url.up.sql

```diff
@@ -0,0 +1,14 @@
+BEGIN;
+
+CREATE TABLE tiny_url (
+  id bigserial PRIMARY KEY,
+  org_url text NOT NULL,
```

Comment:
- what is `org_url` ?

Reply:
original url

Reply:
- please change it to `original_url` to make the column name more descriptive.

Reply:
done